backend/routes/courses.js

```diff
@@ -3,24 +3,84 @@ const courses = require('../logic/courses/courses');
 const router = express.Router();
 
 /*
-* @api {get} /getCourseByTag
-* @apiDescription get list of courses that match a tag
-* @apiParam (query) {string} tag
-* @apiExample {curl} Example usage: GET /courses/getCourseByTag?tag=engineering
+ * @api {get} /getCourseByTag
+ * @apiDescription get list of courses that match a tag
+ * @apiParam (query) {string} tag
+ * @apiExample {curl} Example usage: GET /courses/getCourseByTag?tag=engineering
+ * @author: Alex Lam
+ */
+router.get('/getCourseByTag', async function(req, res, next) {
+  try {
+    let tag = req.query.tag;
+    let result = await courses.queryCourseByTag(tag);
+    res.send(result);
+  } catch (error) {
+    console.log(error);
+    res.status(500).send(error.message);
+  }
+});
+
+/*
+* @api {post} /addCourses
+* @apiDescription This endpoint will add an admin and an associated user
```

Comment:
Is this api info correct?

Reply:
API information is updated to the newest now